addons/project/models/project_task_recurrence.py

```diff
@@ -179,7 +179,7 @@ def _get_next_recurring_dates(self, date_start, repeat_interval, repeat_unit, re
                 start = date_start - relativedelta(days=1)
                 if repeat_type == 'until' and repeat_until > date_start:
                     delta = relativedelta(repeat_until, date_start)
-                    count = delta.years * 12 + delta.months
+                    count = int((delta.years * 12 + delta.months) / repeat_interval)
```

Comment:
Using math.floor or math.ceil would make it clear what the expected behavior is.
I am actually unsure what int(float) makes 🙂 
Would you mind changing it?
Thanks!

Reply:
As long as the commit message is concerned, I kinda agree that knowing the bug we are generating `Reccuring Tashs`, but I think it would be best to write `Reccuring Tas**k**s` 😉